pkg/api/documentschema/documentschema.proto

```diff
@@ -297,4 +308,13 @@ service DocumentService {
     };
   }
 
+  rpc DocumentInsertMany(DocumentBulkInsertRequest) returns (DocumentBulkInsertResponse) {
```

Comment:
if method is named `DocumentInsertMany` it may be better to name messages `DocumentInsertManyRequest` and `DocumentInsertManyResponse`